reactjs-films-homework/src/components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable array-callback-return */
+import React, { useState, useEffect } from 'react';
+import img from './img/Movie-card.png';
+import styles from './MovieCard.module.scss';
+import mockGenres from '../../mocks/genresIDs';
+
+export interface ICard {
+  props: IMovieCard;
+}
+
+export interface IMovieCard {
+  id: number;
+  title: string;
+  vote_average: number;
+  poster_path: string;
+  genre_ids?: number[];
+  key?: number;
+}
+
+export interface IGenre {
+  genres?: {
+    id: number;
+    name: string;
+  }[];
+}
+
+const MovieCard: React.FC<ICard> = ({ props }) => {
+  const { id, title, vote_average, poster_path, genre_ids } = props;
+  const movieTitle = 'Movie Title';
+  const urlImg = `https://image.tmdb.org/t/p/w500/${poster_path}`;
+  const [genre, setGenre] = useState<IGenre>({});
+  const [genreName, setGenreName] = useState<Array<string>>([]);
+
+  mockGenres.then((res) => {
+    return setGenre(res);
+  });
```

Comment:
This logic should be done inside useEffect hook

Reply:
this expression was removed